larpandoracontent/LArObjects/LArDiscreteProbabilityVector.h

```diff
@@ -0,0 +1,393 @@
+/**
+ *  @file   larpandoracontent/LArObjects/LArDiscreteProbabilityVector.h
+ *
+ *  @brief  Header file for the lar discrete probability vector class
+ *
+ *  $Log: $
+ */
+#ifndef LAR_DISCRETE_PROBABILITY_VECTOR_H
+#define LAR_DISCRETE_PROBABILITY_VECTOR_H 1
+
+#include "Pandora/StatusCodes.h"
+
+#include <limits>
+#include <random>
+#include <utility>
+#include <vector>
```

Comment:
Double check all still needed in this header.

Reply:
I think they are
`limits` is needed for epsilon.
`random` is needed for shuffle and mtSomethingSomething
`utility` is needed for `std::pair`
`vector` is needed for... you know ;)

The two pandora headers are needed for `FloatVector` and the status codes.
Some headers may implicitly include other headers but it's better to be explicit here?